Assets/HoloToolkit/Sharing/Scripts/Spawning/SyncSpawnArray.cs

```diff
@@ -33,16 +27,17 @@ protected override T CreateObject(ObjectElement objectElement)
             Type typeToInstantiate = Type.GetType(objectType);
             if (typeToInstantiate == null)
             {
-                Debug.LogErrorFormat("Could not find the SyncModel type to instantiate.");
+                Debug.LogError("Could not find the SyncModel type to instantiate.");
                 return null;
             }
 
-            System.Object createdObject = Activator.CreateInstance(typeToInstantiate);
+            object createdObject = Activator.CreateInstance(typeToInstantiate);
 
-            T spawnedDataModel = createdObject as T;
+            T spawnedDataModel = (T)createdObject;
             spawnedDataModel.Element = objectElement;
             spawnedDataModel.FieldName = objectElement.GetName();
-            spawnedDataModel.Owner = this.Owner; // TODO: this should really come from objectElement.GetOwnerID().  It not safe to assume that this class and its child have the same owner
+            // TODO: this should not query SharingStage, but instead query the underlying session layer
```

Comment:
There's a [place in SyncArray that](https://github.com/Microsoft/HoloToolkit-Unity/blob/master/Assets/HoloToolkit/Sharing/Scripts/SyncModel/SyncArray.cs#L76) that looked very similar and the TODO is from copying this over.

I thought getting the owner this way made more sense than just setting this.Owner.